tests/system/action/poll/test_create.py

```diff
@@ -62,10 +63,13 @@ def test_create_correct(self) -> None:
         assert option.get("text") == "test2"
         assert option.get("poll_id") == 1
         assert option.get("meeting_id") == 1
+        assert option.get("yes") == "10.000000"
         global_option = self.get_model("option/2")
```

Comment:
A hint, you don't need to change it here:
We have a `self.assert_model_exists` - method for reading and asserting the values.

Reply:
Done